.github/workflows/build_conda_linux.yml

```diff
@@ -164,31 +164,36 @@ jobs:
           BINARY_NAME=$(ls "$CONDA_OUTPUT_PATH"/*.tar.bz2)
           echo "$BINARY_NAME"
 
+          export CONDA_ENV_SMOKE="${RUNNER_TEMP}/pytorch_pkg_helpers_${GITHUB_RUN_ID}_smoke"
+          conda create --yes --prefix "${CONDA_ENV_SMOKE}" python="${PYTHON_VERSION}"
+          export CONDA_RUN_SMOKE="conda run -p ${CONDA_ENV_SMOKE}"
+
           if [[ "${{ inputs.package-name }}" = "torchvision" ]]; then
-              ${CONDA_RUN} conda install -y 'numpy>=1.11'
-              ${CONDA_RUN} conda install pillow
+              ${CONDA_RUN_SMOKE} conda install -y 'numpy>=1.11'
+              ${CONDA_RUN_SMOKE} conda install pillow
           fi
           if [[ "${{ inputs.package-name }}" = "torchdata" ]]; then
-              ${CONDA_RUN} conda install -y portalocker>=2.0.0
+              ${CONDA_RUN_SMOKE} conda install -y portalocker>=2.0.0
           fi
 
           if [[ "${GPU_ARCH}" = "cuda" ]]; then
-            ${CONDA_RUN} conda install -v -y -c pytorch-"${CHANNEL}" -c nvidia pytorch pytorch-cuda="${GPU_ARCH_VERSION}"
+            ${CONDA_RUN_SMOKE} conda install -v -y -c pytorch-"${CHANNEL}" -c nvidia pytorch pytorch-cuda="${GPU_ARCH_VERSION}"
           else
-            ${CONDA_RUN} conda install -v -y -c pytorch-"${CHANNEL}" pytorch cpuonly
+            ${CONDA_RUN_SMOKE} conda install -v -y -c pytorch-"${CHANNEL}" pytorch cpuonly
           fi
+          ${CONDA_RUN_SMOKE} conda install  -v -y -c file://"${PWD}/${CONDA_OUTPUT_PATH}" "${{ inputs.package-name }}"
```

Comment:
```suggestion
          ${CONDA_RUN_SMOKE} conda install  -v -y -c pytorch-"${CHANNEL}" --use-local  "${PWD}/${CONDA_OUTPUT_PATH}/${{ inputs.package-name }}*"
```

Reply:
I tried this but it was not working, can try again if you like